website/docs/r/synthetics_monitor.html.markdown

```diff
@@ -3,114 +3,188 @@ layout: "newrelic"
 page_title: "New Relic: newrelic_synthetics_monitor"
 sidebar_current: "docs-newrelic-resource-synthetics-monitor"
 description: |-
-  Create and manage a Synthetics monitor in New Relic.
+Create and manage a Synthetics monitor in New Relic.
 ---
 
 # Resource: newrelic\_synthetics\_monitor
 
-Use this resource to create, update, and delete a synthetics monitor in New Relic.
+Use this resource to create, update, and delete a Simple or Browser Synthetics Monitor in New Relic.
+
+-> **NOTE:** The [newrelic_synthetics_private_location](newrelic_synthetics_private_location.html) resource private minion can take upto 10 minutes to be available through Terraform.
 
 ## Example Usage
 
 ##### Type: `SIMPLE`
+
 ```hcl
 resource "newrelic_synthetics_monitor" "foo" {
-  name = "foo"
-  type = "SIMPLE"
-  frequency = 5
-  status = "ENABLED"
-  locations = ["AWS_US_EAST_1", "AWS_US_EAST_2"]
-
-  uri                       = "https://example.com"               # Required for type "SIMPLE" and "BROWSER"
-  validation_string         = "add example validation check here" # Optional for type "SIMPLE" and "BROWSER"
-  verify_ssl                = true                                # Optional for type "SIMPLE" and "BROWSER"
+  custom_header{
+    name  =  "Name"
+    value = "simpleMonitor"
+  }
+  treat_redirect_as_failure = true
+  validation_string = "success"
+  bypass_head_request = true
+  verify_ssl  = true
+  location_public = ["AP_SOUTH_1"]
+  name  = "%[1]s"
+  period =  "EVERY_MINUTE"
+  status  = "ENABLED"
+  type  = "SIMPLE"
+  tag{
+    key = "monitor"
+    values  = ["myMonitor"]
+  }
+  uri = "https://www.one.newrelic.com"
 }
 ```
 See additional [examples](#additional-examples).
 
-## Argument Reference
+```hcl
+resource "newrelic_synthetics_private_location" "bar1" {
+  description               = "Test Description-Updated"
+  name                      = "%[1]S"
+  verified_script_execution = true
+}
+resource "newrelic_synthetics_monitor" "foo" {
+  custom_header{
```

Comment:
Not a blocker, but we should get your set up with a Terraform HCL formatter for your IDE so we have consistent code style for our examples.